client/state/action-types.js

```diff
@@ -401,6 +401,10 @@ export const PLUGIN_SETUP_INSTRUCTIONS_RECEIVE = 'PLUGIN_SETUP_INSTRUCTIONS_RECE
 export const PLUGIN_UPDATE_REQUEST = 'PLUGINS_UPDATE_REQUEST';
 export const PLUGIN_UPDATE_REQUEST_FAILURE = 'PLUGIN_UPDATE_REQUEST_FAILURE';
 export const PLUGIN_UPDATE_REQUEST_SUCCESS = 'PLUGIN_UPDATE_REQUEST_SUCCESS';
+export const PLUGIN_UPLOAD = 'PLUGIN_UPLOAD';
+export const PLUGIN_UPLOAD_UPDATE_PROGRESS = 'PLUGIN_UPLOAD_UPDATE_PROGRESS';
+export const PLUGIN_UPLOAD_COMPLETE = 'PLUGIN_UPLOAD_COMPLETE';
```

Comment:
I've noticed we tend to use `SUCCESS` and `FAILURE` for naming these usually, do you think it makes sense in this context?